dactyl-config.yml

```diff
@@ -155,7 +155,11 @@ pages:
             - local
             - ripple.com
 
-    # "Transaction Processing" goes here (broken off from transaction reference)
+    -   md: concept-transactions.md
+        category: Technical Overview
```

Comment:
Reiterating my comments on the PR that got closed here, i.e. https://github.com/ripple/ripple-dev-portal/pull/313#discussion_r173554576

Basically, I think we should simplify this organizational scheme to be more similar to @jhaaaa's IA. Otherwise, we are changing the structure of the site on users twice in short order. I propose revising the larger categories to be something like:

* Concepts
    * XRP Ledger Intro
    * XRP Ledger Consensus
    * Reaching Consensus
    * Accounts
    * Crypto Keys
    * Fees
    * Fee Voting
    * Amendments
        * Known Amendments
    * Money in the XRP Ledger
    * Reserves
    * Transaction Cost
    * Partial Payments
    * Escrow
    * Checks
    * Deposit Authorization
    * Transfer Fees
    * NoRipple Flag
    * Authorized Trust Lines
    * Paths
    * Demurrage
    * History Sharding
    * Issuing and Operational Addresses
    * Autobridging GB
    * Historical Ledger Migration GB
* Reference
    * rippled APIs
        * Currency Formats
        * Transactions
        * Data Formats
    * rippled Server
        * Stand-Alone Mode
        * Build and Run rippled on Ubuntu 
    * RippleAPI 
    * Ripple Data API v2
* Tutorials
    * RippleAPI Beginner's Guide
    * Reliable Transaction Submission
    * H/t Multi-Sign
    * Escrow 
    * Payment Channels
    * Working with Regular Key Pair
    * Freezing Issued Currency
    * Listing XRP as an Exchange
    * Becoming an XRP Ledger Gateway
    * Websocket Tool
    * Data API Tool

That could probably stand some tweaking to work out a few of the trickier parts (e.g., should the build instructions be in Reference or Tutorials? Is the nesting level right for Transactions and Data Formats?), but using that structure will probably make the transition to the new Dev Portal a lot easier on our users.